browser/brave_profile_prefs.cc

```diff
@@ -493,6 +494,8 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
                                 base::Value(false));
 
   webcompat_reporter::prefs::RegisterProfilePrefs(registry);
+
+  email_aliases::RegisterProfilePrefs(registry);
```

Comment:
what's the use of these prefs here for? I'm curious if you think it would be useful to integrity protect them to prevent filesystem tampering here

Reply:
Looking at what's set in here, yeah let's integrity protect them: https://github.com/brave/brave-core/pull/27127/files#diff-398d0bffbf3c145bfd429720571b80c331b23abf384570fc0403a493bae06a9aR19-R23

The way to do this is by using a "tracked preference" - https://source.chromium.org/chromium/chromium/src/+/main:services/preferences/tracked/

Reply:
@kdenhartog is that different from a secure preference? Just wondering how integrity protection would help here.